js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -789,6 +818,21 @@ const AssetGraphExplorerWithData = ({
   return explorer;
 };
 
+interface KeyboardTagProps {
+  tooltip?: boolean;
```

Comment:
You'll want to use `$tooltip` to avoid the prop being passed along to the div itself, which would cause console warnings. I might also suggest `$withinTooltip` or something like that, in order to clarify what the prop indicates. (Otherwise it's a little confusing -- does it *have* a tooltip, etc?)

Reply:
got it. TY! Just pushed up a change to resolve this.